interview-questions.md

```diff
@@ -11,58 +11,46 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 
   Researched answer:
 
-
-
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer: git is a programming language. The difference between git and Github is that git is a programming language and Github is a free open source platform for code.
 
+  Researched answer: Git is a tool used for source code management. It can handle small to large projects and is free and open source. Github is a code hosting platform that allows people to work on projects together from anywhere.
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer: I believe a ! will return a Boolean value.
 
+  Researched answer: A logical operator will return a Boolean value.
```

Comment:
There are 2 other operators that will return a Boolean value.  The Relational (<, >, <=, >=) and Equality (==, ===) Operators.